ci-operator/config/openshift/kubecsr/openshift-kubecsr-openshift-4.0.yaml

```diff
@@ -0,0 +1,41 @@
+tag_specification:
+  name: '4.0'
+  namespace: ocp
+promotion:
+  name: '4.0'
+  namespace: ocp
+base_images:
+  base:
+    name: '4.0'
+    namespace: ocp
+    tag: base
+canonical_go_repository: github.com/coreos/kubecsr
```

Comment:
Is this how the code is imported in Go? Or is the repo actually under `coreos` org?

Reply:
the repo is openshift org, the code is imported as above.

Reply:
similar situation to etcd